dxr/plugins/rust/tests/test_mods/test_fns.py

```diff
@@ -0,0 +1,55 @@
+from dxr.testing import DxrInstanceTestCase
```

Comment:
No, there are two test files here - one testing the mods themselves, one testing fns within mods.